### ISSUE TITLE
Using SKLearn Score function

### ISSUE THREAD

**Issue by d-jokic** [BME1478H/Fall2022class#530]
When I try to use the SKLearn score function to see the correlation between the predicted and actual life expectancies, I get an error about needing to reshape the arrays:

<img width="308" alt="image" src="https://user-images.githubusercontent.com/22248638/161793488-d4a14747-9ac7-44fa-bb58-9ba57b74b181.png">

When I reshape the array using array.reshape(-1,1), I get the following error:

<img width="344" alt="image" src="https://user-images.githubusercontent.com/22248638/161793651-0e622afd-287f-4891-9b5b-7ab8420e135f.png">

I'm not sure what the issue is. Here is my code:

    from sklearn.model_selection import train_test_split # keep this here
    X_train, X_test, Y_train, Y_test = train_test_split(X_scaled,Y, test_size=0.2,random_state=0)
      
    # keep this code here
    # confirm the size of the train set and test set make sense
    print(X_train.shape, X_test.shape)
      
    from sklearn.tree import DecisionTreeRegressor # keep this code here
    # create an instance of the Decision Tree Regressor
    tree_model = DecisionTreeRegressor(random_state=0)
      
    # fit the model
    tree_model.fit(X_train, Y_train)
      
    # predictions
    Y_pred = tree_model.predict(X_test)
      
    # keep this code here
    print(Y_pred.shape)
      
    corr_tree = tree_model.score(Y_test, Y_pred)

**Comment by pganga5506** [comment]
The score function automatically calculates Y_pred and compares it with Y_test, but it takes X_test and Y_test as arguments. So using X_test and Y_test inside the score function worked for me.
